.github/workflows/project-sync-pr-labels.yml

```
@@ -0,0 +1,27 @@
+    - cron: '0 3 * * 1' # Every Monday at 3 AM UTC
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  sync-pr-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)